marekventur/wortopia-node
# Update "grunt-angular-templates" to version ~1.0.1

1.0.1 / 2016-01-18
==================

  * 1.0.1
  * Update readme for 1.0.1
  * Merge pull request [#139](https://github.com/ericclemmons/grunt-angular-templates/issues/139) from hvdb/106
    fix: (logging): adjusted logging to error instead of warning when min…

1.0.0 / 2016-01-18
==================

  * Update readme for v1.0.0
  * 1.0.0
  * Merge pull request [#143](https://github.com/ericclemmons/grunt-angular-templates/issues/143) from evilaliv3/improved-unittests
    Improved unittests
  * Enable travis feature for npm packages caching
    This will speed up builds and will lower the travisCI overhead :)
    reference: https://docs.travis-ci.com/user/caching
  * Switch unittests to travis container-based infrastructure
    reference: https://docs.travis-ci.com/user/workers/container-based-infrastructure/
  * Switch unittests from node 0.10 to 0.12 and add 4.2 (LTS)
  * Update test/expected/usemin.html following lates usemin replacements
  * Update dependencies to latest versions

0.6.0 / 2016-01-15
==================

  * Release 0.6.0
  * Merge pull request [#142](https://github.com/ericclemmons/grunt-angular-templates/issues/142) from underscorebrody/single-quotes
    Adds an option to use single quotes instead of double for quote wrapping
  * Adds an option to use single quotes instead of double for quote wrapping
    - We are now doing manual quote wrapping instead of relying on
    JSON.stringify for greater flexibility
